src/components/PetitionBody/index.js

```diff
@@ -3,7 +3,7 @@ import settings from 'settings';
 import Heading2 from 'components/Heading2';
 import Paragraph from 'components/Paragraph';
 
-const PetitionBody = ({ description, suggestedSolution }) => (
+const PetitionBody = ({ description, suggestedSolution, cityResponse }) => (
```

Comment:
not sure if we should break this down into nested containers for

```
<PetitionDescription>
<PetitionSuggestedSolution>
<PetitionCityResponse>"
```

with a shared 'dumb' component for rendering headline / text?

think we're okay for now but any more and we should look to simplify.